### PR TITLE
Gradio Web Demo

### PR DESCRIPTION
**Pull request by AK391**
easy to use web demo on gradio hub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ By [Xiaoyu Xiang](https://engineering.purdue.edu/people/xiaoyu.xiang.1)
 
 ![teaser demo](demos/vinland_saga.gif)
 
+## Gradio Web Demo
+- [Web Demo](https://gradio.app/g/AK391/Anime2Sketch)
+
 ## Updates
 - 2021.5.2: Upload more example results of anime video.
```
Comment (Mukosame) [inline README.md:12]: Thanks @AK391 ! Feel free to add this contribution to ##Updates with your name :)

File: README.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ By [Xiaoyu Xiang](https://engineering.purdue.edu/people/xiaoyu.xiang.1)
 
 ![teaser demo](demos/vinland_saga.gif)
 
+## Gradio Web Demo
+- [Web Demo](https://gradio.app/g/AK391/Anime2Sketch)
```
Comment (Mukosame) [inline README.md]: You can also put a screenshot of the demo page as well as your name and link proudly, like this: https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blame/master/README.md#L31

File: gradiodemo.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import torch
+
+torch.hub.download_url_to_file('https://upload.wikimedia.org/wikipedia/commons/thumb/a/a5/Tsunami_by_hokusai_19th_century.jpg/1920px-Tsunami_by_hokusai_19th_century.jpg', 'wave.jpg')
+torch.hub.download_url_to_file('https://cdn.pixabay.com/photo/2020/10/02/13/49/bridge-5621201_1280.jpg', 'building.jpg')
+
+torchtext.utils.download_from_url("https://drive.google.com/uc?id=1RILKwUdjjBBngB17JHwhZNBEaW4Mr-Ml", root="./weights/")
+
+def sketch2anime(img, load_size=512, gpu_ids=[]):
+    model = create_model(gpu_ids)
+    img, aus_resize = read_img_path(img.name, load_size)
```
Comment (Mukosame) [inline gradiodemo.py:20]: Shall we add "model.eval()"?

**Comment by AK391** [comment]
@Mukosame thanks I have made the suggested updates

**Comment by Mukosame** [comment]
Many thanks! Just merged :D

**Comment by AK391** [comment]
@Mukosame thanks

**Pull request merged**